news/1585.feature

```diff
@@ -0,0 +1 @@
+A new --resolve CLI flag can now be used in conjunction with the --cfg flag to resolve interpolations before printing the config.
```

Comment:
News fragment should complete the sentence:
"This change will ...". See [this](https://hydra.cc/docs/development/documentation#contents-of-a-news-entry).

Reply:
Got it. Made the change in https://github.com/facebookresearch/hydra/pull/1616/commits/57021e390d3a11b58d69ec67321724e84c6f6676.